apps/i18n/signup/en_us.json

```diff
@@ -15,8 +15,11 @@
   "sign_up_as_a_teacher": "Sign up as a teacher",
   "free_curriculum_forever": "Free curriculum. Forever.",
   "read_our_commitment": "[Read our commitment]({commitmentLink}) to keeping our curriculum offerings free for everyone.",
+  "read_our_commitment_free": "Read our commitment to keeping our curriculum offerings free for everyone.",
```

Comment:
I kept the original string to avoid deletion complications.

Reply:
I don't think we're using any of these strings yet, so it's probably safe to felete